src/dotnet/Program.cs

```diff
@@ -260,16 +261,13 @@ private static void PrintVersion()
 
         private static void PrintInfo()
         {
-            HelpCommand.PrintVersionHeader();
-
             DotnetVersionFile versionFile = DotnetFiles.VersionFileObject;
             var commitSha = versionFile.CommitSha ?? "N/A";
+            Reporter.Output.WriteLine($"{LocalizableStrings.DotNetSdkInfoLabel}");
+            Reporter.Output.WriteLine($" Version:   {Product.Version}");
+            Reporter.Output.WriteLine($" Commit:    {commitSha}");
             Reporter.Output.WriteLine();
-            Reporter.Output.WriteLine("Product Information:");
-            Reporter.Output.WriteLine($" Version:            {Product.Version}");
-            Reporter.Output.WriteLine($" Commit SHA-1 hash:  {commitSha}");
-            Reporter.Output.WriteLine();
-            Reporter.Output.WriteLine("Runtime Environment:");
+            Reporter.Output.WriteLine($"{LocalizableStrings.DotNetRuntimeInfoLabel}");
```

Comment:
Adding a pointer to "outdated" but still relevant comment: https://github.com/dotnet/cli/pull/8736#discussion_r172927524

Reply:
@nguerrera wrote:
>> Base Path also reflects global.json and should probably go under .NET Core SDK.

@livarcocc wrote:
> For this, I need @KathleenDollard's agreement. While is makes sense because of the reflecting global.json message, we have always kept this information here.